erpnext/accounts/party.py

```diff
@@ -308,3 +308,13 @@ def set_taxes(party, party_type, posting_date, company, customer_group=None, sup
 		args.update({"use_for_shopping_cart": use_for_shopping_cart})
 
 	return get_tax_template(posting_date, args)
+
+def validate_party_frozen_disabled(party_name, party_type):
```

Comment:
aah! last one for the sake of convention: `def validate_party_frozen_disabled(party_type, party_name):`